bin/environment.php

```diff
@@ -49,4 +49,9 @@
 
 define('SPRYK_ROOT_DIR', __DIR__ . '/../');
 
+$applicationEnv = getenv('APPLICATION_ENV');
+$allowedApplicationEnvs = ['dev', 'prod', 'test'];
```

Comment:
Consider moving these to (class?) constants

Reply:
Where would you put those too?